libs/designsystem/src/lib/components/data-table/table/table.component.ts

```diff
@@ -0,0 +1,11 @@
+import { Component } from '@angular/core';
+
+@Component({
+  // eslint-disable-next-line @angular-eslint/component-selector
+  selector: 'table[kirby-table]',
+  template: `<ng-content></ng-content>`,
+  styleUrls: ['./table.component.scss'],
```

Comment:
We should add OnPush change detection strategy for the table components we decide to keep as components.

Reply:
Is that the best solution when the component doesn't directly contain the data. What does this do for us, when we don't feed a [model] into the data-table?

Reply:
You are right, should have no impact when we have no bindings. Would just be great to have it introduced already before we start implementing further. 

Not everyone will be aware that they need to use this strategy if they add bindings down the road, or they might just forget!

Reply:
Alright, sounds good then :)